lib/jing.rb

```diff
@@ -129,13 +128,18 @@ def execute(options)
   def parse_output(output)
     errors = []
     output.split("\n").each do |line|
-      if line =~ /\A(.+):(\d+):(\d+):\s+\w+:\s+(.+)\Z/
+      case line
+      when /\A(.+):(\d+):(\d+):\s+\w+:\s+(.+)\Z/
         errors << {
           :source  => $1,
           :line    => $2.to_i,
           :column  => $3.to_i,
           :message => $4
         }
+      when /Picked up _JAVA_OPTIONS: /
+        # ignore diagnostic message
+      else # There must have been a problem that was not schema related
+        raise ExecutionError, line
```

Comment:
Maybe we should provide `output` instead of `line` so the exception does not exclude any additional lines/context?

Reply:
Agree, make sens